src/components/ui/sliding-number.tsx

```
@@ -0,0 +1,122 @@
+'use client';
+import { useEffect, useId } from 'react';
+import {
+  MotionValue,
+  motion,
+  useSpring,
+  useTransform,
+  motionValue,
```

ESLint

[error] Member 'motionValue' of the import declaration should be sorted alphabetically.

(sort-imports)